src/app/layouts/global/Footer.tsx

```diff
@@ -1,11 +1,315 @@
 import { useTypedTranslation } from "@shared";
 
-export default function Footer (): JSX.Element {
+export default function Footer(): JSX.Element {
   const t = useTypedTranslation();
 
   return (
-    <footer className="p-3 bg-[#D9D9D9]">
-      <h2>{t("footer")}</h2>
+    <footer className="grid grid-cols-3 divide-x divide-gray-400 bg-background-footer px-12 pt-8 font-rubik">
```

Comment:
we have some font styles. May we create @layer in tailwind (in index.scss) with them? 
![image](https://github.com/Vladyslav-Samokryk/gift-hub-frontend/assets/97731216/67fe2a6a-ef7c-4262-879c-ccbe21f7a96d)